src/main/java/com/conveyal/osmlib/VanillaExtract.java

```diff
@@ -105,11 +105,14 @@ public void service(Request request, Response response) throws Exception {
             response.setContentType("application/osm");
             String uri = request.getDecodedRequestURI();
             LOG.info("VEX request: {}", uri);
-            int suffixIndex = uri.lastIndexOf('.');
-            String fileType = uri.substring(suffixIndex);
             OutputStream outStream = response.getOutputStream();
             try {
+                int suffixIndex = uri.lastIndexOf('.');
+                String fileType = uri.substring(suffixIndex);
                 String[] coords = uri.substring(1, suffixIndex).split("[,;]");
+                if (coords.length < 4) {
+                    throw new IllegalArgumentException("Must provide a bounding box with 4 coordinates");
```

Comment:
This exception should probably be thrown, but there is a TODO in the code below that talks about future work on an extra `buildings` param. Maybe that should just be removed for now?